File: 프로그래머스/0/181925. 수 조작하기 2/수 조작하기 2.js
function oper(num) {
    switch(num) {
        case 1:
            return "w";
        case -1:
            return "s";
        case 10:
            return "d";
        case -10:
            return "a"
    }
}

function solution(numLog) {
    var answer = "";
    
    for(let i = 1; i < numLog.length; i++) {
        answer += oper(numLog[i] - numLog[i-1])
    }
    
    return answer;
}

File: 프로그래머스/0/181925. 수 조작하기 2/수 조작하기 2.py
def solution(numLog):
    log = {
        1: "w",
        -1: "s",
        10: "d",
        -10: "a"
    }
    answer = ""
    
    for i in range(1, len(numLog)):
        answer += log[numLog[i] - numLog[i - 1]]    
    
    return answer